inc/taskjobview.class.php

```diff
@@ -46,6 +46,10 @@ class PluginGlpiinventoryTaskjobView extends PluginGlpiinventoryCommonView
      */
     public function __construct()
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+        $base_url = $CFG_GLPI['url_base'];
+
```

Comment:
Not used (but if a fix must be done, it's probably in `getBaseUrlFor()` method).